### ISSUE TITLE
Mobile: Accessibility: It is difficult to add tags on the tag association screen in landscape orientation

### ISSUE THREAD

**Issue by mrjo118** [laurent22/joplin#12953]
### Operating system

Android

### Joplin version

3.4.4

### Current behaviour

When tapping the search bar on the tag association screen while in landscape orientation, text must be entered in the full screen UI if using the on screen keyboard (due to disableFullscreenUI being set to false by default).

<img width="1817" height="837" alt="Image" src="https://github.com/user-attachments/assets/05f2a7d5-171b-4dc9-b0f6-894b7f899bd9" />

This full screen UI covers the search result, which means you have to press either the tick button on the keyboard or the done button to submit the text. However, when pressing either of those options, it invokes the onSubmit function in ComboBox.tsx, which results in a new tag being created if there is not matching result.

This is undesirable because the user has not had an opportunity to see the search results yet, due to the full screen UI covering it. It is possible to disable the fullscreen UI in landscape mode, by setting disableFullscreenUI to true on the TextInput, but this results in a very squashed tag association screen which is unusable.

<img width="1815" height="855" alt="Image" src="https://github.com/user-attachments/assets/5da65f3d-4d15-4fbe-ab78-d845f92295b6" />

It is possible to enter text in landscape orientation without submitting the value, if the device back button is pressed, but this is not possible on iOS and is not very accessible on Android

### Expected behaviour

Searching for tags on the tag association screen in landscape mode should be more usable. A potential resolution for this would be to continue using the fullscreen UI in landscape mode, but make the onSubmit function not do anything when the device is in landscape mode except to focus on either the first search result or the add new tag entry if there is no match

**Comment by mrjo118** [comment]
Please note, according to https://www.ifelsething.com/post/disable-full-screen-text-input-keyboard/ the utilisation of the full screen ui may be dependent on the available space above the keyboard, rather than the orientation. I have not tested this, but potentially the full screen ui does not get utilised on a tablet in landscape made, due to having a larger screen. So this may need to be considered for the solution used to fix this behaviour

EDIT: I think the full screen ui may be an Android only thing, though I don't have an iOS device to verify this. If this only happens in Android, then it would be best to disable the full screen ui altogether and instead make the window resize and be scrollable in a way that is usable on a small device screen in landscape mode

**Comment by personalizedrefrigerator** [comment]
Thank you for reporting this!

#13028 will allow the tag dialog to scroll vertically. However, it doesn't currently set `disableFullscreenUI={false}`. 

> I have not tested this, but potentially the full screen ui does not get utilised on a tablet in landscape made

The fullscreen UI is not shown on my Android 13 tablet in landscape mode.

**Comment by personalizedrefrigerator** [comment]
Re-opening for now, as per this comment: 

> https://github.com/laurent22/joplin/pull/13028 will allow the tag dialog to scroll vertically. However, it doesn't currently set disableFullscreenUI={false}.
